packages/frontend/src/components/Button/style.module.scss

```diff
@@ -1,9 +1,9 @@
 .button {
   align-items: center;
-  background-color: transparent;
+  background-color: var(--buttonSecondaryBackground);
   border-radius: 10px;
-  border: 0;
-  color: var(--buttonText);
+  color: var(--buttonSecondaryText);
+  border: 1px solid #ccc;
```

Comment:
And the same from "go back from Archive".

![image](https://github.com/user-attachments/assets/edb563c2-2c26-45f1-8c15-f9ac82edbcb1)

I recommend going through all the references of the Button component and seeing how they look.